src/frontend/tsx/Chessboard.tsx

```diff
@@ -345,8 +356,10 @@ export function Chessboard () {
         </p>
       </div>
       <div className='board-container'>
-        <div ref={boardRef} className='chessboard cg-wrap' />
-        { (promotionMove !== null) ? renderPromotionInterface() : <div/> }
+        <div className={`${boardTheme} ${pieceTheme}`}>
```

Comment:
I think this is the correct way to go. The next step is to add an interface for the user to select which board and piece theme he wants and code to inject the appropriate class name into the `div` accordingly. If you want help with this, let me know. Otherwise, I give you free reign creatively, and we can discuss differences of opinion (if any exist) once the code is out.

Reply:
I expect to figure out how to get an interface for theme selection going. The code already has examples of using React for these things. There’s already a <Settings /> component, that I assume I’ll be putting this in. I’ll probably stick close to how lichess.org does it.